### ISSUE TITLE
Merkaartor and IE8 Compatibility View

### ISSUE THREAD

**Issue by tomhughes** [tomhughes/trac-tickets#1799]
**[Submitted to the original trac issue database at 1.22pm, Thursday, 7th May 2009]**

When trying to run Merkaartor with MSIE8 with Compatibility View is OFF (default settings), it doesn't show any track (GPX or downloaded, no matter), only Yahoo background and "red dots". Turning Compatibility View to "Display all websites in Compatibility View" solves a problem. 

This should probably noted somewhere, or fixed.

**Comment by tomhughes** [comment]
**Author: Koying**
**[Added to the original trac issue at 1.41pm, Thursday, 7th May 2009]**

???

You mean Potlach, do you?

Merkaartor do not use MSIE in any way...

**Comment by tomhughes** [comment]
**Author: stro@railways.dp.ua**
**[Added to the original trac issue at 2.23pm, Thursday, 7th May 2009]**

Ah, I see. It should be coincidental when I tried to turn compatibility on and zoomed a track. It really has nothing with IE8. Actually it's a zooming problem, sorry for misleading. 

This is what actually happens: I'm using Vista SP1 and tried 0.13.2, 0.13.1-pre1 and 0.13.1-pre2 -- these versions work fine on XP and Win2003 with IE7, that why I thought about IE8. After trying to be more attentive, I found that when I open GPX in 0.13.2, it zooms viewport to zero (status bar shows -0.1622,0.1619,-0.1622,0.1619), thus I cannot see anything, when I right-click on track and select "Zoom", it zooms to normal size and then works ok. 

I'm attaching three screenshots to show what I've got at start, after open and after zoom.
